testsuite/features/srv_content_lifecycle.feature

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) 2019 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Content lifecycle management 
+
+  Scenario: I create a content lifecycle project Given I am authorized as "admin" with password "admin"
```

Comment:
1. No "I" in scenario titles

2. The "Given" is probably by accident on this line.

Reply:
fixed.